install.sh

```diff
@@ -81,6 +88,32 @@ install_nvm_from_git() {
   return
 }
 
+#
+# Automatically install Node.js
+#
+install_node() {
+  local NODE_VERSION
+  NODE_VERSION=$(node_version)
+
+  if [ -z "$NODE_VERSION" ]; then
+    echo "=> You can now install Node.js by running \`nvm install\`"
+    return 0
+  fi
+
+  echo "=> Installing Node.js version $NODE_VERSION"
+  nvm install $NODE_VERSION
+  nvm use $NODE_VERSION
+  local CURRENT_NVM_NODE
+
+  CURRENT_NVM_NODE="$(nvm_version current)"
+  if [ "$(nvm_version $NODE_VERSION)" == "$CURRENT_NVM_NODE" ]; then
+    nvm alias default "$NODE_VERSION"
```

Comment:
this is also now not required - the first time a version is successfully installed, it's automatically set as the default.